src/compo/modis_aod2ioda.py

```diff
@@ -41,46 +42,67 @@
 
 # Get the group names we use the most.
 metaDataName = iconv.MetaDataName()
-obsValName = iconv.OvalName()
-obsErrName = iconv.OerrName()
-qcName = iconv.OqcName()
 
+varsKeyList = [('valKey', iconv.OvalName(), 'float', 'longitude latitude', '1'),
+               ('errKey', iconv.OerrName(), 'float', 'longitude latitude', '1'),
+               ('qcKey', iconv.OqcName(), 'integer', 'longitude latitude', None)]
+
+float_missing_value = nc.default_fillvals['f4']
+int_missing_value = nc.default_fillvals['i4']
+double_missing_value = nc.default_fillvals['f8']
 long_missing_value = nc.default_fillvals['i8']
+string_missing_value = '_'
```

Comment:
The iconv package contains routines that provide default fill values:
```
float_missing_value = iconv.get_default_fill_val(np.float32)
double_missing_value = iconv.get_default_fill_val(np.float64)
int_missing_value = iconv.get_default_fill_val(np.int32)
long_missing_value = iconv.get_default_fill_val(np.int64)
```
For the numeric types `iconv.get_default_fill_val` returns the corresponding netCDF default values, plus for string types `iconv.get_default_fill_val` returns a null byte. This should work with the string type:
```
string_missing_value = a = iconv.get_default_fill_val(np.str_)
```

I recommend using `iconv.get_default_fill_val` since it covers a superset of the data types in netCDF (most notably a DateTime object), plus if someone ever decides to make a change, this script will automatically pick that up.

Reply:
@srherbener Thank you! I will update them.

Reply:
Sounds great. Thanks. I just noticed I made a copy-paste error in my code example above for the string missing value. The intermediate ` a = ` shouldn't be there:
```
string_missing_value = iconv.get_default_fill_val(np.str_)
```